Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp

```diff
@@ -213,19 +213,19 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian(
                     // x component on x-faces of grid
-                    M_xface(i, j, k, 0) += dt * (PhysConst::mu0 * mag_gammaL) * (M_xface_old(i, j, k, 1) * Hz_eff - M_xface_old(i, j, k, 2) * Hy_eff)
-                                         + dt * Gil_damp * (M_xface_old(i, j, k, 1) * (M_xface_old(i, j, k, 0) * Hy_eff - M_xface_old(i, j, k, 1) * Hx_eff)
-                                         - M_xface_old(i, j, k, 2) * (M_xface_old(i, j, k, 2) * Hx_eff - M_xface_old(i, j, k, 0) * Hz_eff));
+                    M_xface(i, j, k, 0) += dt * (PhysConst::mu0 * mag_gammaL) * (M_old_xface(i, j, k, 1) * Hz_eff - M_old_xface(i, j, k, 2) * Hy_eff)
+                                         + dt * Gil_damp * (M_old_xface(i, j, k, 1) * (M_old_xface(i, j, k, 0) * Hy_eff - M_old_xface(i, j, k, 1) * Hx_eff)
+                                         - M_old_xface(i, j, k, 2) * (M_old_xface(i, j, k, 2) * Hx_eff - M_old_xface(i, j, k, 0) * Hz_eff));
 
                     // y component on x-faces of grid
-                    M_xface(i, j, k, 1) += dt * (PhysConst::mu0 * mag_gammaL) * (M_xface_old(i, j, k, 2) * Hx_eff - M_xface_old(i, j, k, 0) * Hz_eff)
-                                         + dt * Gil_damp * (M_xface_old(i, j, k, 2) * (M_xface_old(i, j, k, 1) * Hz_eff - M_xface_old(i, j, k, 2) * Hy_eff)
-                                         - M_xface_old(i, j, k, 0) * (M_xface_old(i, j, k, 0) * Hy_eff - M_xface_old(i, j, k, 1) * Hx_eff));
+                    M_xface(i, j, k, 1) += dt * (PhysConst::mu0 * mag_gammaL) * (M_old_xface(i, j, k, 2) * Hx_eff - M_old_xface(i, j, k, 0) * Hz_eff)
+                                         + dt * Gil_damp * (M_old_xface(i, j, k, 2) * (M_old_xface(i, j, k, 1) * Hz_eff - M_old_xface(i, j, k, 2) * Hy_eff)
+                                         - M_old_xface(i, j, k, 0) * (M_old_xface(i, j, k, 0) * Hy_eff - M_old_xface(i, j, k, 1) * Hx_eff));
 
                     // z component on x-faces of grid
-                    M_xface(i, j, k, 2) += dt * (PhysConst::mu0 * mag_gammaL) * (M_xface_old(i, j, k, 0) * Hy_eff - M_xface_old(i, j, k, 1) * Hx_eff)
-                                         + dt * Gil_damp * (M_xface_old(i, j, k, 0) * (M_xface_old(i, j, k, 2) * Hx_eff - M_xface_old(i, j, k, 0) * Hz_eff)
-                                         - M_xface_old(i, j, k, 1) * (M_xface_old(i, j, k, 1) * Hz_eff - M_xface_old(i, j, k, 2) * Hy_eff));
+                    M_xface(i, j, k, 2) += dt * (PhysConst::mu0 * mag_gammaL) * (M_old_xface(i, j, k, 0) * Hy_eff - M_old_xface(i, j, k, 1) * Hx_eff)
+                                         + dt * Gil_damp * (M_old_xface(i, j, k, 0) * (M_old_xface(i, j, k, 2) * Hx_eff - M_old_xface(i, j, k, 0) * Hz_eff)
+                                         - M_old_xface(i, j, k, 1) * (M_old_xface(i, j, k, 1) * Hz_eff - M_old_xface(i, j, k, 2) * Hy_eff));
```

Comment:
These changes are to make the notation consistent with MacroscopicEvolveHM_2nd, where we have `_prev` and `_old` right after `M` and before `_xface`. I would like to keep them consistent between these two functions, and it's much simpler to fix this function than the 2nd-order one.